### DISCUSSION TITLE
What does FIX API mean in the docs?

### DISCUSSION THREAD

**Discussion by abhisheksoni27** [nautechsystems/nautilus_trader#1809] [Q&A]
<img width="715" alt="image" src="https://github.com/user-attachments/assets/40bd49cd-4d46-475e-b693-11558054d37b">

**Comment by StumpCore** [comment]
FIX API is a globally designed and standardized format for providing and receiving trade data. To use the FIX API both counterparties (ex. Banks and Brokers) need to setup a FIX connection. While offering many benefits (speed, readability and format) it is linked to additional cost to use those FIX connections. In most cases, institutional players use them but you can use them as well. Interactive Broker offers the FIX connection api for "normal" clients as well.

https://www.onixs.biz/fix-protocol.html
https://www.interactivebrokers.com/campus/ibkr-api-page/fix/

Cost Example from the IB Page:
Minimum equity under management: 10,000 USD
Minimum commissions per month: 1,500 USD
